docs/usage.md

```diff
@@ -32,7 +32,8 @@ Patches::Config.configure do |config|
   config.hipchat_options = {
     api_token: ENV['HIPCHAT_TOKEN'],
     room: ENV['HIPCHAT_ROOM'],
-    user: ENV['HIPCHAT_USERNAME'] #maximum of 15 characters
+    user: ENV['HIPCHAT_USERNAME'], # maximum of 15 characters
+    api_version: 'v1', # optional
```

Comment:
would it be better to clarify what it would default to if no option is provided?

Reply:
In this case, we delegate that to the hipchat gem which currently default to v2, but we're not actually setting it by default in patches.